flask_lambda.py

```diff
@@ -90,6 +91,8 @@ def start_response(self, status, response_headers, exc_info=None):
 
 class FlaskLambda(Flask):
     def __call__(self, event, context):
+        if event.get('isBase64Encoded', False):
+            event['body'] = base64.b64decode(event['body'])
```

Comment:
These lines should prob ably go after the `if 'httpMethod'...` block, so that we are only doing this when in lambda, and not doing it before we make that determination.

Reply:
@sivel Fair comment, I've moved this under the httpMethod condition block so it will only be caught if we're in Lambda.